client/src/entry/analysis/router.js

```diff
@@ -614,6 +615,11 @@ export function getRouter(Galaxy) {
                         component: CredentialsManagement,
                         redirect: redirectAnon(),
                     },
+                    {
+                        path: "user/oidc-profile",
+                        component: UserOidcProfile,
+                        redirect: redirectAnon(),
```

Comment:
```suggestion
                        redirect: redirectIf(!Galaxy.config.enable_oidc || Galaxy.config.enable_account_interface || !hasSingleOidcProfile(Galaxy.config.oidc), "/user") || redirectAnon(),
```

Related to what I've mentioned in [my other comment](https://github.com/galaxyproject/galaxy/pull/21356/files#r2613501539), by doing this ^, this route is inaccessible given this condition.

Of course you'd also need to import
```ts
import { hasSingleOidcProfile } from "@/components/User/ExternalIdentities/ExternalIDHelper";
```

at the top.